docs/CHANGELOG.asciidoc

```diff
@@ -8,16 +8,3934 @@
+The `total` field in `/agent_status` Fleet API responses is deprecated. For more information, refer to {kibana-pull}151564[#151564].
+
+*Impact* +
+The `/agent_status` Fleet API now returns the following statuses:
+
+* `all` &mdash; All active and inactive
+* `active` &mdash; All active
+====
+      
+[discrete]
+[[deprecation-149506]]
+.Deprecates Elastic Synthetics integration
+[%collapsible]
+====
+*Details* +
+The Elastic Synthetics integration is deprecated. For more information, refer to {kibana-pull}149506[#149506].
+
+*Impact* +
+To monitor endpoints, pages, and user journeys, go to **{observability}** -> **Synthetics (beta)**. 
+====
+      
+[float]
+[[features-8.8.0]]
+=== Features
+{kib} 8.8.0 adds the following new and notable features.
+
+Alerting::
+* Adds Maintenance Window Task Runner Integration + New AAD/Event Log Fields {kibana-pull}154761[#154761]
+* Adds support for file attachments in Cases {kibana-pull}154436[#154436]
+* Adds support for users authenticated with API keys to manage alerting rules {kibana-pull}154189[#154189]
+* Adds the ability to control allowed attached file mime types and the maximum file size {kibana-pull}154013[#154013]
+* Adds query and timeframe params to RuleAction to filter alerts {kibana-pull}152360[#152360]
+* Adds the Cases column to the alerts table {kibana-pull}150963[#150963]
+* Adds filtering and sorting for the case activity {kibana-pull}149396[#149396]
+
+APM::
+* Adds group-by feature in APM rules {kibana-pull}155001[#155001]
+* Adds queues as nodes to the service map {kibana-pull}153784[#153784]
+* Adds the ability to display the latest agent version in agent explorer {kibana-pull}153643[#153643]
+* Adds table tabs showing summary of metrics {kibana-pull}153044[#153044]
+* Adds warning to Edit Rule Flyout when publicUrl is not configured {kibana-pull}149832[#149832]
+
+Dashboard::
+Adds the ability to scroll the panel section {kibana-pull}145628[#145628]
+
+Elastic Security::
+For the Elastic Security 8.8.0 release information, refer to {security-guide}/release-notes.html[_Elastic Security Solution Release Notes_].
+
+Enterprise Search::
+For the Elastic Enterprise Search 8.8.0 release information, refer to {enterprise-search-ref}/changelog.html[_Elastic Enterprise Search Documentation Release notes_].
+
+Fleet::
+* Adds audit logging for core CRUD operations {kibana-pull}152118[#152118]
+* Adds modal to display versions changelog {kibana-pull}152082[#152082]
+
+Infrastructure::
+* Adds the logs tab to the Hosts View {kibana-pull}152995[#152995]
+* Adds Alerts tab into Hosts View {kibana-pull}149579[#149579]
+* Adds refactoring to the Time and Position log stream state {kibana-pull}149052[#149052]
+
+Machine Learning::
+* Adds support for custom URLs in jobs for Data Frame Analytics {kibana-pull}154287[#154287]
+* Adds support to filter fields from grouping in Explain Log Rate Spikes{kibana-pull}153864[#153864]
+* Adds log pattern analysis in Discover {kibana-pull}153449[#153449]
```

Comment:
The log pattern analysis would be a better fit in Discover, since it's only surfacing there, I think ... but of course it's also big news for Machine Learning, distributing their awesomeness in more parts of Kibana

Reply:
In the release blog, it lives under Machine Learning. But I don't have any strong opinions. Up to you and ML!

Reply:
let's go with the release blog, we don't adorn ourselve with borrowed plumes 🪶

Reply:
Can it be added to both Discover and Machine Learning? If not, then I agree with @kertal that it is better in Discover.

Reply:
it's a big thing for both, so it would be worth a duplicate ... of we create a new section like **Disco Machine**  🕺🎰🤖💃

Reply:
New Disco Machine section. ON IT.

Reply:
Just kidding. I added it to both sections: ML and Discover.

Reply:
Thx, **Disco🕺🎰  Lens 🤖💃 Machine** ... it will be in 9.0, FYI @ninoslavmiskovic because of the upcoming product branding !